code/modules/food_and_drinks/food/foods/meat.dm

```diff
@@ -79,7 +79,7 @@
 			"<span class ='notice'>[user] cuts the raw cutlet with [W]!</span>", \
 			"<span class ='notice'>You cut the raw cutlet with [W]!</span>" \
 			)
-		new /obj/item/reagent_containers/food/snacks/raw_bacon(loc)
+		new /obj/item/reagent_containers/food/snacks/raw_bacon(src)
```

Comment:
can you try cutting this on a table? I have a feeling this is going to cause raw_bacon to get instantiated in the contents of the rawcutlet

Reply:
It also creates raw bacon if cut on the table. Taking a look at the raw cutlet variables before it gets deleted doesn't seem to show anything weird. Looking at the variable of the subsequent raw bacon also doesn't seem to show anything weird to me. Is there something else I should be looking for in regards to this issue?

Reply:
alright cool, I just wasn't sure if passing `src` was gonna cause weird behaviour here or not.

Reply:
Passing `src` here will absolutely instantiate the bacon in the contents of the cutlet, which is something to avoid. Maybe it still spawns on the table because the cutlet is deleted and so it spills it's contents? Not sure about this right now, need to investigate.

Reply:
Maybe just get_turf(src)?

Reply:
Aye, I was thinking something like:
```dm
var/obj/item/reagent_containers/food/snacks/raw_bacon/bacon = new(get_turf(src))
if(ishuman(loc))
    var/mob/living/carbon/human/H = loc
    H.put_in_hands(bacon)
```
`new(get_turf(src))` so it always spawns on the floor no matter what. But if the cutlet's loc was a human, try to put it in their hands.

Reply:
Done!